docs/source/callbacks.rst

```diff
@@ -105,6 +105,7 @@ Lightning has a few built-in callbacks.
     ModelCheckpoint
     ProgressBar
     ProgressBarBase
+    ModelPruning
```

Comment:
this should probably be moved up if we want to keep it in alphabetical order

Reply:
Done !